concourse/pipelines/daisy-build.yaml

```diff
@@ -0,0 +1,190 @@
+---
+resource_types:
+- name: gcs
+  type: registry-image
+  source:
+    repository: frodenas/gcs-resource
+
+resources:
+# Git repos
+- name: compute-daisy
+  type: git
+  source:
+    uri: https://github.com/hopkiw/compute-daisy.git
+    branch: dockerfile
+- name: compute-image-tools
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/compute-image-tools.git
+    branch: master
+    paths:
+    - daisy_workflows/**
+
+# Output artifacts
+- name: linux-executable
+  type: gcs
+  source:
+    bucket: compute-image-tools-test
+    json_key: |
+      ((concourse-sa.credential))
+    versioned_file: release/linux/daisy
+- name: windows-executable
+  type: gcs
+  source:
+    bucket: compute-image-tools-test
+    json_key: |
+      ((concourse-sa.credential))
+    versioned_file: release/windows/daisy
+- name: darwin-executable
+  type: gcs
+  source:
+    bucket: compute-image-tools-test
+    json_key: |
+      ((concourse-sa.credential))
```

Comment:
i don't think this is the right secret name? it looks in the template to be `((gcs-key.credential))`